### PR TITLE
feat: show meeting requests beyond rg/wg groups

### LINKED ISSUE #3986: The session requests page needs to show non wg/rg requests
We are now using Sessions to model meetings of many group types other than rg/wg. These sessions should show on the /meeting/requests page.

### PR DESCRIPTION
**Pull request by jennifer-richards**
This shows "meeting requests" for groups that are not rg/wg groups. These requests are modeled as requests for sessions.

Previously, only sessions for groups that had an area group as their parent were listed. Now any group that is of a type that `has_meetings` is shown, but `admin` and `social` sessions are suppressed. The latter criterion is applied to prevent the various breaks / registration sessions / etc that make up a meeting from swamping the request list. 

Fixes #3986.

### CODE REVIEW

File: ietf/meeting/tests_views.py [hunk 1]
```diff
@@ -6398,17 +6398,81 @@ class SessionTests(TestCase):
...
+        # parse the HTML
+        self.assertContains(r, has_meetings_admin_session.group.acronym)
+        self.assertNotContains(r, _sreq_edit_link(has_meetings_admin_session))  # no link to the session request
+        self.assertContains(r, has_meetings_social_session.group.acronym)
+        self.assertNotContains(r, _sreq_edit_link(has_meetings_social_session))  # no link to the session request
         self.assertContains(r, requested_session.constraints().first().name)
         self.assertContains(r, conflicting_session.group.acronym)
+        self.assertContains(r, has_meetings.group.acronym)
+        self.assertContains(r, has_meetings_not_meeting.group.acronym)
+        self.assertNotContains(r, not_has_meetings.group.acronym)
```
Comment (meadmaker) [inline ietf/meeting/tests_views.py]: Do you not care whether these meetings have a link to their session requests?

**Review by meadmaker — APPROVED** [review]
Merge when you're ready.

File: ietf/meeting/tests_views.py (hunk 1 above)
Comment (jennifer-richards) [reply ietf/meeting/tests_views.py]: Oh, actually I do - I'd forgotten that they have links shown. Thanks!

**Comment by jennifer-richards** [comment]
Closing this, will submit upstream PR

**Pull request closed without merging**